WeatherSource.cs

```diff
@@ -92,8 +103,19 @@ public static IEnumerator UpdateCloudRenderTex ()
 
 				OnCloudRendered();
 
```

Comment:
I think you're missing something. This should and—verified via logging that I've since removed—does recompute the rain spawn textures once for every 16 times the shadow texture is recomputed.

Reply:
You're right, makes sense. The coroutine doesn't break out of the for loop til i=16 is reached.